projects/ngx-prx-styleguide/src/assets/styles/stories/button.stories.ts

```diff
@@ -0,0 +1,207 @@
+import { storiesOf, moduleMetadata } from '@storybook/angular';
+import { centered } from '@storybook/addon-centered/angular';
+
+// Module metadata for stories.
+const storiesModuleMetaData = moduleMetadata({
+  imports: [],
+  schemas: [],
+  declarations: [],
+  providers: [],
+});
+
+storiesOf('Global|CSS', module)
+  .addDecorator(centered)
+  .addDecorator(storiesModuleMetaData)
+  .add(
+    'Button',
+    () => ({
+      styleUrls: [
+        '../../../../../../src/styles.css'
+      ],
+      template: `
+        <section class="main demo">
```

Comment:
Yes, that is the plan. I only continued using the demo css and markup here since these stories are acting as a test bed for tweaks to legacy css. In the future I want to flush out the stories for the global CSS to be a little more targeted as things are refactored.